src/actions/Button.scss

```diff
@@ -141,25 +141,25 @@
     }
   }
 
-  &[data-variant="highlight"] {
-    background-color: var(--seeds-color-highlight);
-    border-color: var(--seeds-color-highlight);
+  &[data-variant="warn"] {
```

Comment:
Updated the naming here to warn - we don't have a highlight in the Figma tokens, and the colors are the same